pkg/client/inspect_image.go

```diff
@@ -52,6 +52,48 @@ type ImageInfo struct {
 	Processes ProcessDetails
 }
 
+// ImageWithExtensionInfo is a collection of metadata describing
+// an app image built with extensions using Cloud Native Buildpacks.
+type ImageWithExtensionInfo struct {
+	// Stack Identifier used when building this image
+	StackID string
+
+	// List of buildpacks that passed detection, ran their build
+	// phases and made a contribution to this image.
+	Buildpacks []buildpack.GroupElement
+
+	// List of extensions that passed detection, ran their generate
+	// phases and made a contribution to this image.
+	Extensions []buildpack.GroupElement
```

Comment:
@itsdarshankumar  I have a doubt, which problems did you face if you just add  `Extensions []buildpack.GroupElement` into `ImageInfo` struct? I mean, it an `ImageInfo` could have extension, then is this new struct totally necessary? did you face issues trying to print the output in that way? I just want to understand why do we need this new one.